modules/deploy/pages/deployment-option/self-hosted/manual/linux-system-tuning.adoc

```diff
@@ -0,0 +1,28 @@
+= Linux System Tuning
+:description: Learn how Redpanda applies automatic tunic to your Linux system.
+
+Redpanda includes numerous features to tune your Linux systems for optimal performance. You can trigger hardware-based optimizations through the `rpk redpanda tune` command, also called the autotuner. Software-based configurations are managed automatically through the `systemd` suite of tools.
+
+== User-triggered hardware tuning
+
+Users invoke the Redpanda autotuner using the `rpk redpanda tune` command. This command may optimize both native Linux and Kubernetes nodes. It analyzes the hardware configuration of the worker node and sets appropriate kernel options. You can choose to set certain hardware parameters using `ulimit` or other system tools while using the autotuner to manage others.
+
+=== Using the autotuner
+
+The Redpanda autotuner has a large array of hardware tuners to choose from. You may choose to either execute all available tuners, or to apply only a subset of them. This is particularly important when you wish to maintain specific configurations and prevent Redpanda from altering these settings.
+
+Redpanda recommends running the autotuner as part of your production deployment workflows. This ensures any hardware configurations or updates to the autotuner itself are incorporated quickly and efficiently into your system.
+
+For the full list of available tuners, refer to the xref:reference:rpk/rpk-redpanda/rpk-redpanda-tune.adoc[`rpk redpanda tune` guide]. The autotuner is also available for managing Kubernetes nodes, but note that changes to the Linux kernel for running Kubernetes images are not persisted. For more information on Kubernetes configuration and execution, refer to xref:deploy:deployment-option/self-hosted/kubernetes/k-tune-workers.adoc[Kubernetes worker node tuning].
+
+== Automated software configurations
+
+Redpanda makes use of systemd to manage operating system and software-based limits, such as memlocks, file handle limits, core limits, and certain distribution-specific configurations. These help ensure the system executes stably and efficiently. Some key examples of automated configurations include:
+
+* Using systemd also offers some support to detect hung services and infinite looping scenarios. This helps ensure you don't have blocking services or processes that may interfere with Redpanda's stream processing.
```

Comment:
We offer a non-systemd version of crash-loop detection internally via https://github.com/redpanda-data/redpanda/issues/6569 as well. Not sure if its worth mentioning here though.

Reply:
Upon re-reading this seems to be talking about non-Redpanda processes running on a node with Redpanda. The crash-loop detection I mentioned above only applies to the Redpanda process its self.